src/display/lv_display.c

```diff
@@ -205,6 +207,8 @@ void lv_display_delete(lv_display_t * disp)
     lv_free(disp);
 
     if(was_default) lv_display_set_default(lv_ll_get_head(disp_ll_p));
+
+    if(was_refr) lv_refr_set_disp_refreshing(lv_ll_get_head(disp_ll_p));
```

Comment:
Setting `NULL` makes more sense to me. Why would we change the being refreshed display to an arbitrary other display?

In the case of `display_default` using another display is fine.